### PR TITLE
CPU Usage: add new module

### PR DESCRIPTION
**Pull request by CarterLi**
It will wait for 1 second, but this is the only sollution I can use because the only interface the kernel provides is `/proc/stat` AFAIK.

It's not enabled by default. If you still can't accept the change, fill free to close this PR.

### CODE REVIEW

**Comment by CarterLi** [comment]
Fix #111

File: presets/verbose [hunk 1]
```diff
@@ -18,6 +18,7 @@
 --terminal-format "Process: {}; Path: {}; Exe: {}"
 --terminal-font-format "Raw: {}; Name: {}; Size: {}; Styles: {}; Pretty: {}"
 --cpu-format "Name: {}; Pretty: {}; Vendor: {}; Logical online: {}; Logical configured: {}; Physical: {}; Cores: {}; bios: {}; scaling max: {}; scaling min: {}; info max: {}; info min: {}; cpuinfo: {}; frequency: {}"
+--cpu-usage-format "Perentage: {}"
```
Comment (LinusDierheimer) [inline presets/verbose:21]: typo

File: src/data/config.txt [hunk 2]
```diff
@@ -124,6 +124,7 @@
 #--terminal-key Terminal
 #--terminal-font-key Terminal Font
 #--cpu-key CPU
+#--cpu-usage-key CPU Usage
```
Comment (LinusDierheimer) [inline src/data/config.txt:127]: Add for --cpu-usage-format to format options too

File: src/data/help.txt [hunk 3]
```diff
@@ -78,6 +78,7 @@ Key options: Provide a custom key for an output
     --terminal-key <key>
     --terminal-font-key <key>
     --cpu-key <key>
+    --cpu-usage-key <key>
```
Comment (LinusDierheimer) [inline src/data/help.txt:81]: Add --cpu-usage-format to format help too

**Review by LinusDierheimer — COMMENTED** [review]
Add cpuusage to src/data/modules.txt, so it is printed with `fastfetch --list-modules`.

**Comment by CarterLi** [comment]
Done @LinusDierheimer

**Comment by LinusDierheimer** [comment]
I think `cpuusage` is still missing from `src/data/modules.txt`

**Comment by CarterLi** [comment]
There are so many of them 🤣

**Comment by LinusDierheimer** [comment]
I admit some documentation would benefit the project, i am just missing the time.

**Pull request merged**